mmv1/third_party/terraform/services/logging/resource_logging_sink.go

```diff
@@ -155,10 +155,11 @@ func expandResourceLoggingSinkForUpdate(d *schema.ResourceData) (sink *logging.L
 		Filter:          d.Get("filter").(string),
 		Disabled:        d.Get("disabled").(bool),
 		Description:     d.Get("description").(string),
-		ForceSendFields: []string{"Destination", "Filter", "Disabled"},
+		Exclusions:      expandLoggingSinkExclusions(d.Get("exclusions")),
```

Comment:
Could you briefly explain the solution you've chosen in this file regarding the treatment of exclusions? By my read, we could be clearing the exclusions when they are not set by the user, which would be a change to the current behavior of a permadiff. Is that correct?

Reply:
Yeah, why do we need to change the functionality around exclusions here?

Reply:
The current provider, when it detects there is an exclusion during a refresh, but no exclusion in the terraform resource, deletes the exclusion. This change ensures that when we acquire the resource the same behavior is applied.